proj.sh

```diff
@@ -3,54 +3,51 @@ export PATH=$PATH:/usr/local/bin
 
 function generate-data {
   echo 'Data generation started.'
-  sh -ac ' . ./.env; yarn --cwd $DATA_GENERATOR_PATH start $DATA_FILES_PATH'
-  sh -ac ' . ./.env; cp -r $DATA_GENERATOR_PATH/data/ ./neo4j/import'
-  echo 'Data generation completed.'
+  sh -ac ' . ./.env && yarn --cwd $DATA_GENERATOR_PATH start $DATA_FILES_PATH "$@" && cp -r $DATA_GENERATOR_PATH/data/ ./neo4j/import'
 }
 
 function build-stack {
   generate-data
-  docker-compose -f docker-compose.yml -f docker-compose-$MET_ATLAS_VERSION.yml build $@
+  docker-compose -f docker-compose.yml -f docker-compose-local.yml build $@
 }
 
 function start-stack {
-  docker-compose -f docker-compose.yml -f docker-compose-$MET_ATLAS_VERSION.yml up -d
+  docker-compose -f docker-compose.yml -f docker-compose-local.yml up -d
 }
 
 function stop-stack {
-  docker-compose -f docker-compose.yml -f docker-compose-$MET_ATLAS_VERSION.yml kill
+  docker-compose -f docker-compose.yml -f docker-compose-local.yml kill
 }
 
 function clean-stack {
   docker stop $(docker ps -a -q) || true
   docker rm $(docker ps -a -q) || true
   docker volume prune --force || true
-  # The line below was not removing the db container properly
-  docker-compose -f docker-compose.yml -f docker-compose-$MET_ATLAS_VERSION.yml down
-  docker volume prune -f
 }
 
 function logs {
-  docker-compose -f docker-compose.yml -f docker-compose-$MET_ATLAS_VERSION.yml logs -f $@
+  docker-compose -f docker-compose.yml -f docker-compose-local.yml logs -f $@
 }
 
 function deploy-stack {
   generate-data
   docker-compose -f docker-compose.yml -f docker-compose-prod.yml --context $1 up -d --build
 }
 
+function import-db {
+  generate-data --drop-indexes
+  source .env
+  docker exec -it neo4j bash -c "cypher-shell -u ${NEO4J_USERNAME} -p ${NEO4J_PASSWORD} --format plain 'MATCH (n) DETACH DELETE n;'"
```

Comment:
These cypher statements could be added to the data-generator as well, when `--drop-indexes` (which probably can be renamed to `--reset-db`) is provided.

Reply:
Great idea!